packages/plugins/src/plugins/aave/getLendingFetcher.ts

```diff
@@ -1,36 +1,59 @@
 import { EvmNetworkIdType, ProxyInfo } from '@sonarwatch/portfolio-core';
 import { Cache } from '../../Cache';
 import { fetchLendingForAddress } from './helpers';
-import { FetcherExecutor } from '../../Fetcher';
-import { lendingConfigs } from './constants';
+import { Fetcher } from '../../Fetcher';
 import { getDSA } from '../../utils/evm/getDSA';
+import { fetchLendingForAddressV3 } from './helpersV3';
+import { LendingConfig } from './types';
+import { combinedLendingConfigs } from './constants';
 
-export default function getLendingFetcherExecutor(
-  networkId: EvmNetworkIdType
-): FetcherExecutor {
-  return async (owner: string, cache: Cache) => {
+export default function getLendingFetcher(
+  networkId: EvmNetworkIdType,
+  platformId: string,
+  version: LendingConfig['version']
+): Fetcher {
+  const executor = async (owner: string, cache: Cache) => {
     const instadappDSA = getDSA(networkId);
     const addresses = [owner];
```

Comment:
```suggestion
    const addresses = [getAddress(owner)];
```

To be similar to https://github.com/Octav-Labs/sonarwatch-portfolio/pull/43 and follow @jtardioli 's recommendation ?

Reply:
The added benefit will be stronger typing in the rest of the modules.